packages/validator/specs/api.json

```diff
@@ -12,7 +12,7 @@
     "__conditions": [
       {
         "__if": {
-          "__this": "(?<={)[^\\/{}]+(?=})"
+          "__this_name": "(?<={)[^\\/{}]+(?=})"
```

Comment:
I think it would be less confusing to name this file exactly the same as the field it refers to (so `apiSpecifications.json` instead of `api.json`)

Reply:
Renamed in 4b9caca